src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
```

Comment:
Prefer using 'const' for 'totalCost' as it is not reassigned. If you need to apply discounts, you can return the adjusted cost directly from the 'if' statements.